src/Controls/src/Core/ServiceScope.cs

```diff
@@ -0,0 +1,12 @@
+namespace Microsoft.Maui.Controls
+{
+	/// <summary>
+	/// Defines the scope of service registration in the DI container.
+	/// </summary>
+	public enum ServiceScope
+	{
+		Singleton,
```

Comment:
Is this necessary? The DI framework already has a ServiceLifetime enum

https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.servicelifetime?view=dotnet-plat-ext-7.0

Reply:
Yes, it serves the purpose. Requires changes in the source gen as it can target only netstandard2.0. But can be managed internally as it only holds the state.

Reply:
Am making changes for including the ServiceLifetime enum as it is the same as the ServiceScope enum.

Need your comments on whether the relevant property is to be updated as `Lifetime` or leave it as `Scope`,  the current one.

Reply:
Have made the changes with the ServiceLifetime enum. Since `Lifetime` as the property name resembled close to its usage, have renamed Scope as well. Hope it's fine.